packages/web-components/src/components/cbp-action-bar/cbp-action-bar.scss

```diff
@@ -0,0 +1,53 @@
+/**
+ * @prop --cbp-action-bar-color-background: var(--cbp-color-grey-cool-70);
+ * @prop--cbp-action-bar-color-background-dark: var(--cbp-color-grey-cool-5);
+*/
+
+:root {
+ --cbp-action-bar-color-background: var(--cbp-color-gray-cool-70);
+ --cbp-action-bar-color-background-dark: var(--cbp-color-gray-cool-5);
+
+ --cbp-action-bar-info-color: var(--cbp-color-text-lightest);
```

Comment:
you can remove the "info" from this. It should just be the color for anything in the action bar. You should have a color and a color-bg (shorten it for consistency).